js/components/scrubberBar.js

```diff
@@ -132,11 +132,13 @@ var ScrubberBar = React.createClass({
       document.removeEventListener("touchend", this.handlePlayheadMouseUp, true);
     }
     this.props.controller.seek(this.props.currentPlayhead);
-    this.setState({
-      currentPlayhead: this.props.currentPlayhead,
-      scrubbingPlayheadX: 0
-    });
-    this.props.controller.endSeeking();
+    if (this.isMounted()) {
```

Comment:
After seeking from the end screen we get to the pause screen and end screen scrubber bar becomes unmounted causing console warning to appear.